google/ggrc-core
# GGRC-3917 Fix task counts

# Issue description

Task counts between top right and the tab on my work page did not match 


# Steps to test the changes

I have not investigated what the differences in tasks is, so I just used ggrc_qa database and a few different users. Any user with the following ID in a line ending with `*` had a missmatch.
So we can log in as these users and check the counts.

```
person:   2 - counts:   180 -   182   *
person:   4 - counts:   180 -   180
person:   5 - counts:   547 -   549   *
person:   6 - counts:   839 -   839
person:   7 - counts:  1488 -  1529   *
person:   8 - counts:   180 -   180
person:   9 - counts:   180 -   180
person:  10 - counts:   180 -   180
person:  11 - counts:  4220 -  4221   *
person:  12 - counts:   180 -   180
person:  13 - counts:   181 -   181
person:  14 - counts:   180 -   180
person:  15 - counts:   180 -   180
person:  17 - counts:   420 -   420
person:  18 - counts:   180 -   180
person:  19 - counts:   181 -   181
person:  20 - counts:   242 -   242
person:  21 - counts:   180 -   180
person:  22 - counts:   260 -   287   *
person:  23 - counts:    90 -    90
person:  30 - counts:     1 -     1
person:  40 - counts:     1 -     1
person:  41 - counts:     4 -     5   *
person:  65 - counts:    13 -    13
person:  95 - counts:     1 -     1
person: 104 - counts:     6 -     6
person: 118 - counts:     1 -     1
person: 124 - counts:     1 -     1
person: 128 - counts:     1 -     1
person: 134 - counts:    26 -    26
person: 135 - counts:  4160 -  4161   *
person: 146 - counts:     1 -     1
person: 150 - counts:     1 -     1
person: 164 - counts:     1 -     1
person: 182 - counts:     2 -     2
person: 186 - counts:   210 -   213   *
person: 192 - counts:     1 -     1
person: 219 - counts:    10 -    10
person: 228 - counts:     1 -     1
person: 229 - counts:  1667 -  1695   *
person: 230 - counts:    42 -    42
person: 234 - counts:   769 -   778   *
person: 235 - counts:     2 -     2
person: 236 - counts:   192 -   192
person: 237 - counts:   337 -   337
person: 239 - counts:     1 -     1
person: 251 - counts:   211 -   213   *
person: 252 - counts:   193 -   193
person: 255 - counts:     4 -     4
person: 256 - counts:   192 -   193   *
person: 259 - counts:   890 -   890
person: 261 - counts:   218 -   218
person: 263 - counts:   878 -   885   *
person: 266 - counts:     1 -     1
person: 267 - counts:   251 -   251
person: 271 - counts:     1 -     1
person: 272 - counts:     1 -     1
person: 275 - counts:     1 -     1
person: 276 - counts:   236 -   236
person: 277 - counts:   243 -   243
person: 278 - counts:   211 -   211
person: 281 - counts:   209 -   209
person: 289 - counts:     1 -     1
person: 292 - counts:   192 -   192
person: 294 - counts:     1 -     1
person: 297 - counts:   192 -   192
person: 298 - counts:     1 -     1
person: 307 - counts:     1 -     1
person: 311 - counts:  5265 -  5453   *
person: 327 - counts:  5445 -  5634   *
person: 328 - counts:  4787 -  5177   *
person: 333 - counts:     1 -     1
person: 339 - counts:  1258 -  1615   *
person: 341 - counts:  1257 -  1426   *
person: 344 - counts:   284 -   285   *
person: 347 - counts:  1254 -  1423   *
person: 350 - counts:   273 -   273
person: 353 - counts:   912 -   912
person: 356 - counts:  1027 -  1028   *
person: 365 - counts:     1 -     1
person: 375 - counts:   904 -   904
person: 380 - counts:   192 -   192
person: 381 - counts:   199 -   199
person: 383 - counts:   205 -   205
person: 390 - counts:    22 -    22
person: 391 - counts:    94 -    94
person: 392 - counts:   194 -   194
person: 393 - counts:  1758 -  1775   *
person: 398 - counts:   194 -   197   *
person: 401 - counts:   186 -   186
person: 402 - counts:  2494 -  2566   *
person: 403 - counts:   800 -   826   *
person: 404 - counts:   461 -   479   *
person: 415 - counts:   180 -   180
person: 417 - counts:   215 -   215
person: 431 - counts:     1 -     1
person: 438 - counts:   180 -   180
person: 441 - counts:   256 -   256
person: 456 - counts:   180 -   180
person: 457 - counts:   180 -   180
person: 464 - counts:     1 -     1
person: 468 - counts:   310 -   312   *
person: 477 - counts:   210 -   210
person: 479 - counts:   180 -   180
person: 480 - counts:   180 -   180
person: 481 - counts:   180 -   180
person: 486 - counts:   180 -   180
person: 493 - counts:   182 -   182
person: 494 - counts:   224 -   228   *
person: 496 - counts:   180 -   180
person: 497 - counts:   181 -   181
person: 499 - counts:   180 -   180
person: 500 - counts:   180 -   180
person: 503 - counts:   164 -   167   *
person: 520 - counts:     7 -     8   *
person: 531 - counts:     1 -     1
person: 1329 - counts:     3 -     3
person: 1572 - counts:     2 -     2
person: 1579 - counts:    11 -    11
```
# Solution description

I did not check what the differences are in the queries, but I just made sure that the same query is used to get the same data. I asked around about what the correct expected count is (to see which was right and which was wrong) but Since I could not find the correct spec on this, I settled for just unifying the data. If we have to modify anything we can now fix only one query and that will work on both places (task count and my work count)

# Sanity checklist




# PR Review checklist
